masterfabric-mobile/swift-camp
# CL:Images at Swift

## 📋 PR Description

This PR introduces a new SwiftUI component, ImageView, which showcases a small Atatürk exhibition with a clean and modern design. The component displays a grid of six images with a descriptive title at the top and a "Go Back" button for navigation.

---

## ✅ Checklist


---

## 🛠 Steps to Test

1. Open the project in Xcode and ensure all dependencies are resolved.
2. Navigate to the ImageView implementation.
3. See the big pictures in the view
4. Click the "1881-∞" button in to go back
5. Run the application and confirm no performance issues or UI 



---

## 🔗 Related Links


- Documentation: https://developer.apple.com/documentation/swiftui/image

---

### 📷 Screenshots (Optional)

<img width="412" alt="11" src="https://github.com/user-attachments/assets/15e2e29a-44c8-4dcc-9ee4-20c3f798929f" />
<img width="412" alt="12" src="https://github.com/user-attachments/assets/fa7c1856-d118-4d10-acda-c29f9781a342" />
